metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py

```diff
@@ -1168,6 +1213,12 @@ def make_lineage_dataset_urn(self, platform: str, name: str) -> str:
             platform, name, platform_instance, self.config.env
         )
 
+    def get_platform_instance_id(self) -> str:
+        return self.config.platform_instance or self.platform
```

Comment:
this isn't needed - instead it should just raise `NotImplementedError`

Reply:
This comments is not clear to me. Can you please be more specific how this method should look like ?

Reply:
From this - https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/state/stateful_ingestion_base.py#L281
looks like, we don't need to add this method in subclass at all. I'll remove this.

Reply:
Done